src/components/Map.js

```diff
@@ -70,7 +71,7 @@ class Map extends Component {
       });
     }
     let bounds = new google.maps.LatLngBounds(); //eslint-disable-line
-    let currentSelectedMarker;
+    //let currentSelectedMarker;
```

Comment:
Thanks for bringing all the variables to the top. This makes it easier to read the code and ensures that whereever we use the variable it will be instantiated already.